packages/dd-trace/src/platform/node/index.js

```diff
@@ -20,6 +20,7 @@ const semver = require('semver')
 const emitter = new EventEmitter()
 
 const hasSupportedAsyncLocalStorage = semver.satisfies(process.versions.node, '>=14.5 || ^12.19.0')
+const hasSupportedAsyncResource = semver.satisfies(process.versions.node, '>= 14 || ^13.9.0 || ^12.19.0')
```

Comment:
Shouldn't these be the exact same versions as `AsyncLocalStorage`?

Reply:
No, because executionAsyncResource was added before AsyncLocalStorage was.

Reply:
It's not really about when it was added, but more about when the bugs affecting us were fixed, which was in https://github.com/nodejs/node/pull/33801 which affected both `AsyncLocalStorage` and `executionAsyncResource` and landed in 14.5.0.